frontend/src/routes/index.tsx

```diff
@@ -5,27 +5,15 @@ import projectRoutes from "./subRoutes/projectRoutes";
 import contractRoutes from "./subRoutes/contractRoutes";
 import adminRoutes from "./subRoutes/adminRoutes";
 import reportRoutes from "./subRoutes/reportRoutes";
-import { Home, Login, PageNotFound, Users } from "../pages";
+import { Home, Login, PageNotFound } from "../pages";
 import { Main } from "../components";
-import useAuthorization from "hooks/useAuthorization";
-import keycloak from "keycloak";
-import { ICurrentUser } from "types";
 import Unauthorized from "pages/Unauthorized";
 
 const AppRouter: FC = () => {
-  const { currentUser } = useAuthorization(keycloak);
   return (
     <Routes>
       <Route element={<Main />}>
-        <Route
-          path="/"
-          element={
-            <AuthorizedRoute
-              currentUserRole={(currentUser as ICurrentUser)?.role_id?.label}
-              allowedRoles={["Administrator", "Manager"]}
-            />
-          }
-        >
+        <Route path="/" element={<AuthorizedRoute allowedRoles={["Administrator", "Manager"]} />}>
```

Comment:
will these still be valid

Reply:
Yes, for now.  This code was a part of the code that need to change completely as we are doing to many auth checks to present data.